Chrysalis-Papilio/Soen-490-Papilio
# User profile page

# General info
The user profile page is ready and can be accessed from the NavBar

**Issue number**: PAPILIO-17 (just frontend)

**Task description**: 
 - Static User Profile page is now in
 - Can be accessed from the Nav Bar
 - Done mostly according to the UI prototypes with a small number of medium sized deviations
 - Features scrollable horizontal list of activities (also static as of yet)

# Manual Testing
To test the User Profile page, you have to:

1. Open the app and either sign up or log in
2. Tap on "Account" on the NavBar
3. Tap the "Profile" button 
4. Now you're on the user profile
5. Now verify that the user profile hides the nav bar (opens it's own separate page)
6. Also verify that the horizontal scrollable list is actually scrollable and that you can scroll it horizontally

# Notes
- This is a purely frontend task, the backend for this will come shortly
- This branch was created and populated with cherry-picked comments from the #14 branch for our efforts to fix branch #14
